vars/filebeat.groovy

```diff
@@ -100,8 +121,9 @@ def stop(Map args = [:]){
     docker exec -t ${stepConfig.id} chmod -R ugo+rw /output || echo "Exit code \$?"
     docker stop --time ${timeout} ${stepConfig.id} || echo "Exit code \$?"
   """)
-  if(archiveOnlyOnFail == false || (archiveOnlyOnFail && isBuildFailure())){
-    archiveArtifacts(artifacts: "**/${stepConfig.output}*", allowEmptyArchive: true)
+
+  if(!archiveOnlyOnFail || isBuildFailure()){
+    archiveArtifacts(artifacts: "${workdir}/${stepConfig.output}*", allowEmptyArchive: true)
```

Comment:
archiveArtifacts only support relative paths, so "${workdir}" should be a relative path to the WORKSPACE, if not this will not archive anything

Reply:
Wdyt about always using `env.WORKSPACE` to store the files?

Reply:
the problem is not to store the likes, right now `filebeat(workdir: "${WORKSPACE}")` will store the files on the WORKSPACE, and the relative path `**/${stepConfig.output}*` will make `archiveArtifacts` to archive the files. 
to Make the proposal change, it needs to change `workdir` to work as a relative path, so instead of `filebeat(workdir: "${WORKSPACE}")` the equivalent step would be `filebeat(workdir: ".")` and the wordir default value would be `.` instead of `pwd()`

Reply:
so we change how the parameter `workdir` works